Libs/MRML/Widgets/qMRMLScalarsDisplayWidget.h

```diff
@@ -0,0 +1,124 @@
+
+/*==============================================================================
+
+  Program: 3D Slicer
+
+  Portions (c) Copyright 2020 Brigham and Women's Hospital (BWH) All Rights Reserved.
+
+  See COPYRIGHT.txt
+  or http://www.slicer.org/copyright/copyright.txt for details.
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+==============================================================================*/
+
+#ifndef __qMRMLScalarsDisplayWidget_h
+#define __qMRMLScalarsDisplayWidget_h
+
+// MRMLWidgets includes
+#include "qMRMLWidget.h"
+
+// CTK includes
+#include <ctkVTKObject.h>
+
+class qMRMLScalarsDisplayWidgetPrivate;
+class vtkMRMLColorNode;
+class vtkMRMLDisplayNode;
+class vtkMRMLNode;
+
+class QMRML_WIDGETS_EXPORT qMRMLScalarsDisplayWidget : public qMRMLWidget
+{
+  Q_OBJECT
+  QVTK_OBJECT
+
+  Q_PROPERTY(ControlMode scalarRangeMode READ scalarRangeMode WRITE setScalarRangeMode)
+  Q_ENUMS(ControlMode)
+
+public:
+  /// Constructors
+  typedef qMRMLWidget Superclass;
+  explicit qMRMLScalarsDisplayWidget(QWidget* parentWidget = nullptr);
+  ~qMRMLScalarsDisplayWidget() override;
+
+  /// Get the (first) current display node
+  vtkMRMLDisplayNode* mrmlDisplayNode()const;
+  /// Get current display nodes (if multi selection)
+  QList<vtkMRMLDisplayNode*> mrmlDisplayNodes()const;
+
+  bool scalarsVisibility()const;
+  QString activeScalarName()const;
+  vtkMRMLColorNode* scalarsColorNode()const;
+
+  enum ControlMode
```

Comment:
I think we should remove this and just use the modes defined in vtkMRMLDisplayNode.